webapp/tokenList/index.ts

```diff
@@ -59,11 +58,7 @@ export const getRemoteTokens = function (token: EvmToken) {
 const hemiTokens: Token[] = (hemilabsTokenList.tokens as EvmToken[])
   .filter(t => t.chainId === hemiMainnet.id || t.chainId === hemiTestnet.id)
   // TODO the following line once bitcoin is enabled https://github.com/hemilabs/ui-monorepo/issues/738
-  .filter(
-    t =>
-      (t.symbol !== 'hemiBTC' && t.symbol !== 'tBTC') ||
-      featureFlags.btcTunnelEnabled,
-  )
+  .filter(t => t.symbol !== 'hemiBTC' && t.symbol !== 'tBTC')
```

Comment:
This should also be removed. 
```suggestion
```

If `featureFlags.btcTunnelEnabled` was true, the `.filter()` callback would always return true, and would only be considered if the feature was disabled. But as now it will be always enabled, the filter needs to be removed.

Reply:
This ☝🏽  causes the app to crash as soon as the user selects Bitcoin, so let's make sure to run a smoke test at least


https://github.com/user-attachments/assets/591082bf-10fa-4f14-ad18-ad84500a75bd

Reply:
removed